### DISCUSSION TITLE
Viewed files reopen when page is refreshed

### DISCUSSION THREAD

**Discussion by PedroCorreiaLuis** [community/community#10123] [Pull Requests]
The title says it all! 

Whenever a file is ticked as Viewed in a PR it collapses it, but if you refresh the page, the ticked toggle remains there, which is a good thing, but the file uncollapses which is ratter a bad thing in my opinion

**Comment by scaramallion** [comment]
It would be nice to see a fix for this, very annoying. You have to untick then click on the checkbox, then click again before it collapses.

**Comment by smoogipoo** [comment]
I've made a ticket for this but it's been unanswered for the past 3 days... This is heavily impacting my review process. Please fix!

**Comment by peppy** [comment]
Can't seem to find any documentation of this being a change/feature, so seems like a bug. Please fix, this makes reviewing very large pull requests a huge drain. Also referenced [here](https://education.github.community/t/viewed-files-in-pr-no-longer-collapsed-on-revisit/72228) by another user.

**Comment by d-edery** [comment]
Please fix. This is very annoying. Reviewing large PRs (hundreds of files) while developers fix stuff on the go is a mess.
The only "Viewed" files that should stay open (uncollapsed) are the ones that has the "Changed since last viewed" on them. This is a good enough reason to keep it open so that I could easily view the changes
